source/chapters/advanced_topics.rst

```diff
@@ -3138,36 +3138,40 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
      ===== ================== ============ ======== ===== ======
      Value Description        Library      Playlist Crate Browse
      ===== ================== ============ ======== ===== ======
-     0     Artist             X            X        X     X
-     1     Title              X            X        X     X
-     2     Album              X            X        X     X
-     3     Albumartist        X            X        X     X
-     4     Year               X            X        X     X
-     5     Genre              X            X        X     X
-     6     Composer           X            X        X     X
-     7     Grouping           X            X        X     X
-     8     Tracknumber        X            X        X     X
-     9     Filetype           X            X        X     X
-     10    Native Location    X            X        X     X
-     11    Comment            X            X        X     X
-     12    Duration           X            X        X     X
-     13    Bitrate            X            X        X     X
-     14    BPM                X            X        X     X
-     15    ReplayGain         X            X        X     X
-     16    Datetime Added     X            X        X     X
-     17    Times Played       X            X        X     X
-     18    Rating             X            X        X     X
-     19    Key                X            X        X     X
-     20    Preview            X            X        X     X
-     21    Coverart           X            X        X
-     22    Position                        X
-     23    Playlist ID                     X
-     24    Location                        X
-     25    Filename                                       X
-     26    File Modified Time                             X
-     27    File Creation Time                             X
+     0     (Current Column)   X            X        X     X
```

Comment:
```suggestion
     0     *Current Column*   X            X        X     X
```

Reply:
I removed Current Column from this table at all, because it's unidirectional, from Controller to Mixxx. But this table documents the [Library],sort_column which always returns the 'real' column id